samples/bluetooth/fast_pair/input_device/sysbuild.cmake

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+if(NOT DEFINED FP_MODEL_ID AND NOT DEFINED FP_ANTI_SPOOFING_KEY)
+  message(WARNING "
+  -------------------------------------------------------
+  --- WARNING: Using demo Fast Pair Model ID and Fast ---
+  --- Pair Anti Spoofing Key, it should not be used   ---
+  --- for production.                                 ---
+  -------------------------------------------------------
+  \n"
+  )
+  set(FP_MODEL_ID "0x2A410B" PARENT_SCOPE)
```

Comment:
Can we use `zephyr_set` here to use Zephyr API consistently for get/set operations?

Reply:
Also, please remember about this one: https://github.com/nrfconnect/sdk-nrf/pull/12958#discussion_r1383080398

Reply:
It makes no sense to use `zephyr_set` here

Reply:
Ok, I will not insist here.